src/main/java/qna/domain/DeleteHistory.java

```diff
@@ -31,6 +31,20 @@ public DeleteHistory(ContentType contentType, Long contentId, User deletedBy, Lo
         this.createDate = createDate;
     }
 
+    private DeleteHistory(ContentType contentType, Long contentId, User deletedBy) {
+        this.contentType = contentType;
+        this.contentId = contentId;
+        this.deletedBy = deletedBy;
+    }
```

Comment:
위에 있는 주생성자를 활용해보시는건 어떨까요?
```suggestion
    private DeleteHistory(ContentType contentType, Long contentId, User deletedBy) {
        this(contentType, contentId, deletedBy, LocalDateTime.now());
    }
```
링크를 참고해보셔도 좋습니다. [생성자 하나를 주 생성자로 만드세요.](https://jackjeong.tistory.com/148)

Reply:
```
메서드가 늘어날수록 SRP를 위반할 수도 있다.
```
생성자 또한 인스턴스화하는 메서드니 SRP를 위반할 수 있는 요소로 작용할 수도 있겠네요 👍 👍 👍